index.html

```diff
@@ -1,54 +1,183 @@
-                </p>
-           </div>
-           <div class="header-date">
-            <h1>2022.04.11(MON) ~ 13(WED)</h1>
-            <p>@iHub Kenya, Caren Shopping Centre.</p>
-           </div>
-       </div>
-   </header>
-   <section class="program">
-       <div class="program-header">
-            <h2>Main Program</h2>
-            <hr>
-       </div>
-       <div class="program-list">
-           <div class="program-list-items">
-               
-           </div>
-       </div>
-   </section>
+  <header class="header-container">
+    <nav class="social">
+      <ul>
+        <li>
+          <a href="https://twitter.com/odhiambo_ed"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/edward-odhiambo-6a462a21b/"><i class="fa-brands fa-linkedin"
+             ></i></a>
+        </li>
+        <li>
+          <a href="https://github.com/white3d"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li><a href="">English</a></li>
+        <li><a href="">French</a></li>
+      </ul>
+    </nav>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="index.html"><img src="./images/logo.png" alt="IDREALITY" /></a>
+      </div>
+      <div class="hamburger">
+        <a href="#"><img class="icon-bar" src="./images/hamburger.png" alt="Icon" /></a>
+      </div>
+      <div class="desktop-nav">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a class="nav-btn" href="#">Share Your Mind</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav id="mobile-nav" class="mobile-nav">
+      <span class="mobile-close"><i class="fas fa-times"></i></span>
+      <ul>
+        <li><a href="index.html" onclick="menuLink()">Home</a></li>
+        <li><a href="about.html" onclick="menuLink()">About</a></li>
+        <li><a href="#" onclick="menuLink()">Blog</a></li>
+        <li><a class="nav-btn" href="#" onclick="menuLink()">Share Your Mind</a></li>
+      </ul>
+    </nav>
+    <div class="header-text">
+      <h3>Hi! You want your ideas turned to reality?</h3>
+      <h1>
+        Idreality is all you need. You've just gotten all you need at one
+        stop.
+      </h1>
+      <div class="border-text">
+        <p>
+          Idreality is an innovation and life coaching hub, a digital platform
+          that provides solutions to the ideas that have not been birthed,
+          taking them from incubation to execution. It is a free online space
+          to accommodate the youth and their ideas, and walk them towards
+          realization of their potential.
+        </p>
+      </div>
+      <div class="header-date">
+        <h2>~iDreality! Where the ideas come to realization~</h2>
+        <p>@iHub Kenya, Caren Shopping Centre.</p>
+      </div>
+    </div>
```

Comment:
The `header-text` `div` should not be included in the `header` as the header should include only the logo and navigation

- [ ] Please ensure that the headline section of your home page and about page is outside of the header;
- [ ]  Please adapt your design from the home page to match the Behance design regarding the spacing and the font-weight
Your design:
![image](https://user-images.githubusercontent.com/83596135/157727356-2e9238bb-d22a-4ab8-950b-63e05056adcc.png)
Behance design:
![image](https://user-images.githubusercontent.com/83596135/157727442-33525703-1027-4375-a83e-315315e9451c.png)